doc/whats_new.rst

```diff
@@ -12,6 +12,56 @@ Version 0.18
 Changelog
 ---------
 
+.. _model_selection_changes:
+
+Model Selection Enhancements and API Changes
+--------------------------------------------
+
+  - **The ``model_selection`` module**
+
+    The classes and functions from the formerly :mod:`sklearn.cross_validation`,
+    :mod:`sklearn.grid_search` and :mod:`sklearn.learning_curve` are
+    grouped together into a single module available at
+    :mod:`sklearn.model_selection`. There are some major changes to these
+    newly moved classes and functions. Refer below to know about the
+    changes.
+
+  - **Data-independent CV splitters enabling nested cross-validation**
+
+    The new cross-validation splitters, defined in the
+    :mod:`sklearn.model_selection`, are not initialized with any
+    data-dependent parameters. Instead they expose a :func:`split` method
+    that takes in the data and yields a generator for the different splits.
+
+    This change makes it possible to do nested cross-validation with ease,
+    facilitated by :class:`model_selection.GridSearchCV` and similar
+    utilities.
+
+  - **The enhanced `results_` attribute**
+
+    The new ``results_`` attribute (of :class:`model_selection.GridSearchCV`
+    and :class:`model_selection.RandomizedSearchCV`) introduced in lieu of the
+    ``grid_scores_`` attribute is a dict of 1D (numpy) (masked) arrays with
```

Comment:
What's with the (numpy) (masked) syntax here, it would not be clear to me why those are in parantheses. Does this sometime return non-numpy arrays?

Maybe it's better to just say 1d arrays, and I would prefer moving the Pandas motivation to the first paragraph, as readers don't always read to the end.